tests/xtro-sharpie/xtro-sanity/Sanitizer.cs

```diff
@@ -198,8 +199,16 @@ public static int Main (string [] args)
 						if (found)
 							break;
 					}
-					if (!found)
+					if (!found) {
 						Log ($"?unknown-entry? {entry} in 'common-{fx}.ignore'");
+						failures.Add (entry);
+					}
+				}
+				if (failures.Count > 0 && !string.IsNullOrEmpty (Environment.GetEnvironmentVariable ("AUTO_SANITIZE"))) {
```

Comment:
This code is nearly identical to lines 232-238, probably good for a new function.

Reply:
Agreed it would be better - but it's a backport PR to minimize merge conflicts during summer and back into master